test-snap-build/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: Test Snap Build
+description: Build the snap locally using Snapcraft and review the output.
+author: Snapcrafters
+branding:
+  icon: zap
+  color: orange
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout the source
+      uses: actions/checkout@v4
+
+    - name: Build snap
+      uses: snapcore/action-build@v1
+      id: build
+
+    - name: Review snap
+      uses: diddlesnaps/snapcraft-review-action@v1
+      with:
+        snap: ${{ steps.build.outputs.snap }}
+        isClassic: 'false'
```

Comment:
This kind of stuff should be an input (including approved plugs and slots), but then the file in the source repo will almost be as complex as this file. So do you still think we should use a remote action/workflow for this?

Reply:
Ah yes - I actually solved this problem for the classic argument in one of the other workflows - it's pretty easy to compute that so I'll bring the same across here.

Will have a think about the plugs/slots declarations too :)